Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -198,7 +198,7 @@ std::vector<u8> NANDImporter::GetEntryData(const NANDFSTEntry& entry)
   return data;
 }
 
-bool NANDImporter::ExtractCertificates()
+bool NANDImporter::ExtractCertificates() const
```

Comment:
This one is questionable. I probably wouldn't make it `const`.
Think about what these functions are doing instead of just making it `const` when the compiler says it can be.

This function changes files on the filesystem.
If it's not just a simple "getter", it probably doesn't make sense being `const`.